tests/basic-request.json

```diff
@@ -15,7 +15,7 @@
             "url": "url 2"
         }
     ],
-    "click_data": [
+    "click_histories": [
```

Comment:
I feel like we've fixed this six times this week already, but...reverting the big dependency management change undid it again 😓